spec/models/course/announcement_spec.rb

```diff
@@ -1,6 +1,9 @@
 require 'rails_helper'
 
 RSpec.describe Course::Announcement, type: :model do
+  it { should belong_to(:creator).class_name(User.name) }
+  it { should belong_to(:course).inverse_of(:announcements) }
```

Comment:
@kxmbrian, I think here need to change all the should syntax to expect http://betterspecs.org/#expect

<code> it { is_expected.to XXX } </code>